portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +32,174 @@ function randomizeImage() {
   imageContainer.innerHTML = '';
   imageContainer.appendChild(imgElement);
 }
+
+/**
+ * Handles response by converting it to text and passing the result to
+ * addNameToDom().
+ */
+function handleResponse(response) {
+  console.log('Handling the response.');
+
+  // response.text() returns a Promise, because the response is a stream of
+  // content and not a simple variable.
+  const textPromise = response.text();
+
+  // When the response is converted to text, pass the result into the
+  // addNameToDom() function.
+  textPromise.then(addNameToDom);
+}
+
+/** Adds a name to the DOM. */
+function addNameToDom(name) {
+  console.log('Adding name to dom: ' + name);
+
+  const NameContainer = document.getElementById('name-container');
```

Comment:
Variable names should be camel case in js.

https://www.w3schools.com/js/js_conventions.asp